### DISCUSSION TITLE
这就是认知战喽~

### DISCUSSION THREAD

**Discussion by Staok** [The-Run-Philosophy-Organization/run#190] [General]
“为何而润.md” 这篇文章典型充满无脑黑（就是为了黑而黑啊），选择性、带有目的的措辞，与台湾1450和美国的反华腔调、言论很像嘛。
不过，我也一直认为，中国的很多问题究其根源，是人口太多、人口素质一般而导致的低效内卷，这是其根源之一。
都润出去也好，减轻国内竞争。。

**Comment by ytgui** [comment]
@Staok 您的好日子还在后面呢
![FK5xovYUYAACLyt](https://user-images.githubusercontent.com/12762888/163835931-e0aaa4b5-a2d2-4e0b-8933-3e2817ba7411.jpeg)

**Reply by leon-fudan** [reply]
这既是run的人的素质吗？

**Reply by xijinping0** [reply]
> 这既是run的人的素质吗？

层主素质哪里有问题？

> @Staok 您的好日子还在后面呢

层主祝你日子越来越好，这正是中国国力不断上升、人民生活水平越来越高的体现。这句话有什么问题？

> 一分钟也很厉害了

层主对于身体素质欠缺的人，也非常鼓励。这体现出层主积极向上的生活态度，和对网友精甚细腻的关怀。

[![精甚细腻](https://img.youtube.com/vi/OBKQUDmWDRo/0.jpg)](https://youtu.be/OBKQUDmWDRo?t=4)

**Comment by sakurashino** [comment]
期待您的高见，写好了麻烦链接发一下

**Comment by the-run-philosophy** [comment]
看起来您的认知确实出了点问题

**Comment by ghost** [comment]
> “为何而润.md” 这篇文章典型充满无脑黑（就是为了黑而黑啊），选择性、带有目的的措辞，与台湾1450和美国的反华腔调、言论很像嘛。 不过，我也一直认为，中国的很多问题究其根源，是人口太多、人口素质一般而导致的低效内卷，这是其根源之一。 都润出去也好，减轻国内竞争。。

请问[为何而润](https://github.com/The-Run-Philosophy-Organization/run/blob/main/%E5%93%B2%E5%AD%A6%E6%A6%82%E5%BF%B5/%E4%B8%BA%E4%BD%95%E8%80%8C%E6%B6%A6.md)这篇文章是哪个部分充满无脑黑？欢迎指出并进行修正

**Reply by supreding** [reply]
我感觉比较扯的两点，
1.美国企业不会被铁拳砸，tiktok确实是成功了，那华为一流为什么至今还吃着美国的铁拳？为什么还有一堆高校被美国实体制裁？说到底不过是利益不相关罢了，没动到美国老爷们的蛋糕
2.说中的政策是自上而下，而外国是自下而上。前者确实没错，但是后者是哪里来的？难道一个美国人就可以参与国家政策的制定吗，掌权的还不是那0.0001%的人？难道我说因为枪击不安全，第二天我提议美国禁枪或者实行更加严格的枪支管理法案就能通过？  还是那一点，没动到老爷们的蛋糕随你怎么蹦跶，动到他们的蛋糕那是万万不行的

说到底，在中国是羊群的人，难道去了外国就不是羊群吗？开头说不是为了钱而run，我认为到不如直接真诚些，在哪里我都是羊，选一个草料多些赚美刀的羊圈，这才是合乎逻辑的选择吧

**Reply by edward8n24isnotavailable** [reply]
哈哈哈薄纱

**Comment by h7221836** [comment]
辩证看待就好

**Comment by leon-fudan** [comment]
> “为何而润.md” 这篇文章典型充满无脑黑（就是为了黑而黑啊），选择性、带有目的的措辞，与台湾1450和美国的反华腔调、言论很像嘛。 不过，我也一直认为，中国的很多问题究其根源，是人口太多、人口素质一般而导致的低效内卷，这是其根源之一。 都润出去也好，减轻国内竞争。。
---------------------------------------
这么一说好像也对，中国的技术人员已经越来越多。也许一些比较容易被西方洗脑的人自己跑出去，也没带走什么东西，最多只是多花一些冤枉申请费用。祝愿他们在外面过得好，不要再回来。中国人加油吧，等着看这些人看清楚了外面的世界，哭着喊着回国的时候再来回顾这个可笑的repo。

**Comment by the-run-philosophy** [comment]
你说的太对了，毕竟不是所有人都喜欢呆在家里被饿死～
中国小学没毕业的人越来越多，也许每天看张维为、金灿荣、新闻联播的人自己决定留下来，对全世界没有什么损失，反而稳固了帝国的燃料，为欧美提供廉价的物资。祝他们在国内越过越好。加油吧，等这些人铁拳砸到自己头上的时候，当自己被关在家里或者被强制拖去方舱医院camp，或者拖着行李步行50公里回家的时候，再哭着喊着想起来要出国赶紧打开这个repo结果发现连网都上不去了。。。。

**Reply by leon-fudan** [reply]
我无法理解，美国100万死在不知道哪里的人，他们怎么会如此默默无闻且无人在意。他们临死前是不是也在努力呼吸最后一口空气而不得？他们是否挣扎呼喊？在死亡面前，一个找不到方舱的人，拖着行李什么的人，你如此关心？

**Comment by NatsuKawaKo** [comment]
然而西方人不会介意负面新闻，甚至就新闻内容还会和你一起抨击政府。
而你这种小粉红看到国内的负面新闻就会像这样着急，开始无脑护主。

**Reply by Staok** [reply]
国内大小媒体和个人批判中国政府的多了去了，我也觉得当下问题百出，所以看到你这句话我就笑了，你这个给洋人跪下的优越感也太强了。如果外国媒体批判中国讲的对，那请加大力度，但是因为有好多都没有说到点上啊（比如说什么独裁政府，讲出这句话的人自己可能都不信），所以我才讲是无脑黑嘛。

**Reply by NatsuKawaKo** [reply]
开始偷换概念了是吧？什么时候说是独裁了？原文不是指少数几十年不滚下台的迂腐之人频频做出糟糕决策，阻碍国家发展吗？有说错吗？教培一刀切几十万人失业，你不是被铁拳砸过的人肯定不会认为这有丝毫问题。

**Reply by walker1996** [reply]
难道你不知道中小学教培一刀切很多平民老百姓拍手称快吗？苦中小学教培很久了。
当然，也砸了不少人饭碗，这也是事实。

**Reply by missing233** [reply]
> 开始偷换概念了是吧？什么时候说是独裁了？原文不是指少数几十年不滚下台的迂腐之人频频做出糟糕决策，阻碍国家发展吗？有说错吗？教培一刀切几十万人失业，你不是被铁拳砸过的人肯定不会认为这有丝毫问题。

推动教育公平有问题吗？还是说你喜欢看平民交不起高昂的课外辅导费后他们的子女也落入底层？

**Comment by Miku-programm** [comment]
哪有新闻自由和言论自由自己心里清楚，其他不多说

**Reply by leon-fudan** [reply]
对，很多给你的平台上，你是有言论自由，但是无人访问。
他们所标榜的言论自由是因为他们控制着传播的霸权，你只有粪坑吐槽自由罢了。

**Reply by cambrian-rock** [reply]
所有的传播背后都有霸权，在外言论自由面对的霸权要考虑的问题是“传播度”和“曝光率”大不大，在内言论自由面对的霸权是“能不能说”“会不会被屏蔽”“会不会文字狱”“会不会小黑屋”

**Reply by leon-fudan** [reply]
川普从小黑屋出来点了个踩

**Comment by heng4719** [comment]
群众是盲目且易被煽动的，真可悲。

**Comment by letsgow** [comment]
run出去我倒是不反对，能不能把从小到大占的公共资源吐出来再走。

**Reply by NatsuKawaKo** [reply]
能不能把吞的税吐出来给我

**Reply by letsgow** [reply]
> 能不能把吞的税吐出来给我

你多少岁？你四十岁有资格这样说，你读书或者才毕业就没资格这样说。别说你爸妈帮你交了，那只代表你爸妈有资格run出去。你一个啥都没有完成的fw也能叫？

**Reply by NatsuKawaKo** [reply]
急了急了，也就你这种低能孤儿能做到和父母精准切割了吧（笑）

**Reply by letsgow** [reply]
> 急了急了，也就你这种低能孤儿能做到和父母精准切割了吧（笑）

急了急了，承认自己是什么也没完成过的小fw了吧，笑~

**Reply by NatsuKawaKo** [reply]
孤儿野种是这样的，没有爸爸妈妈就觉得国家是无偿送公共资源给他的，从事实来看确实如此

**Reply by letsgow** [reply]
果然，只是让你做一个独立的人就这么难吗？说到底你自己到底做了什么贡献啊？我说了你40了是有资格说退税啊？智商不行？还是个小fw~要是润出去还是个小fw，哈哈

**Reply by NatsuKawaKo** [reply]
没贡献就不能跑，没交税就是fw，就你这脑瘫逻辑还在这狗吠？不如赶紧去认个爹妈吧小耶种

**Reply by AntEffect** [reply]
> 没贡献就不能跑，没交税就是fw，就你这脑瘫逻辑还在这狗吠？不如赶紧去认个爹妈吧小耶种

哈哈，这货是典型工具人人格，不当狗不舒服

**Reply by missing233** [reply]
> 急了急了，也就你这种低能孤儿能做到和父母精准切割了吧（笑）

最先急的似乎只有你一个

**Comment by Winter-And-You-Gone** [comment]
纯纯的扯淡，在这种地方宣传这种东西，不就是反共吗？总以为自己懂得很多，接触了一些西方所谓自由主义思潮觉得大彻大悟了，不用搞这么多名词，也不用新建这么多新词，直说你们反共就好了，你要润我们不拦着，但是你在这扭曲、臆想、歪曲、搞虚无主义，在这种网站宣传涉及政治的东西，不是蠢，是很明显的坏，西方的嘴脸自己看不清楚被洗脑了还要来祸害别人捏

**Reply by Staok** [reply]
什么是“主观”，什么是“断章取义”、“春秋笔法”，什么是“愚民媒体”，什么是“选择性写作”，这些词不需要我解释吧。贼喊捉贼的把戏司空见惯，流氓言行也太阳底下不是什么新鲜事。

**Reply by xijinping0** [reply]
> 在这种地方宣传这种东西，不就是反共吗？

对啊，反共，怎么啦？不爱看你别来啊。

**Reply by Winter-And-You-Gone** [reply]
> > 在这种地方宣传这种东西，不就是反共吗？
> 
> 对啊，反共，怎么啦？不爱看你别来啊。

笑死，你承认就好，粪坑还是得来的，不然去哪吐痰，正好这是个智商筛选器，把你们集中起来也挺好

**Reply by xijinping0** [reply]
来突开化粪池是吧？果然和习主席一样啊。

[![突开](https://img.youtube.com/vi/B8V4rrKuTkE/0.jpg)](https://youtu.be/B8V4rrKuTkE?t=113)

《[习近平与他的情人们](https://lovers.xijinping.one)》
https://github.com/xijinping0

**Comment by CKwuliuqi** [comment]
这有啥好讨论的，看看710大捷，抢了你的钱，还派人打你，还要你磕头认错，就问你服不服！

**Comment by MarionNoel** [comment]
润太祖可是特大人命事故入狱的局长公子，贪二代，有钱的大老板。他大发善心，关爱新一代it耗材，安排咱们世世代代跟着他，给他做贡献。
https://github.com/MarionNoel/juzhanggongzi/discussions/
